yaml/buildings.yaml

```diff
@@ -18,6 +18,12 @@ globals:
       roof_orientation: {col: "roof:orientation"}
       addr_housenumber: {col: "addr:housenumber"}
       addr_street: {col: "addr:street"}
+  - &building_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail(building)
+  - &building_part_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail("building:part")
```

Comment:
Some of these (`dome`, `part`, `main`, `floor`) seem like they could be useful for rendering, although I don't know a huge amount about the 2.5D building/part stuff.

Values like `historic` and `proposed` make me a bit nervous - it sounds like they may be applied to buildings which aren't currently present in reality. Might we want to give it a different kind to help make it clearer to someone using the data that it's different and, for example, might overlap with other buildings or features which pre- or post-date the historic / proposed / collapsed / ruined -building?